.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - image: circleci/python:3.6
     working_directory: ~/st2-packages
     environment:
-      DISTROS: "xenial bionic el7 el8"
+      DISTROS: "xenial bionic focal el7 el8"
```

Comment:
You want to adjust the parallelism in CircleCI from `4` to `5` above.
Otherwise `el8` is excluded from testing and building.

Reply:
I didn't realise parallelism actually meant truncate. :shocked: Thanks for pointing that out, parallelism has been bumped to 5.